src/main/java/com/yelp/nrtsearch/server/utils/ParallelDocumentIndexer.java

```diff
@@ -86,10 +87,8 @@
                     oneDocBuilder, rawLines.stream(), luceneServerClient));
         return genIdFuture;
       } catch (RejectedExecutionException e) {
-        logger.log(
-            Level.WARNING,
-            String.format("Waiting for 1s for LinkedBlockingQueue to have more capacity"),
-            e);
+        logger.warn(
+            String.format("Waiting for 1s for LinkedBlockingQueue to have more capacity"), e);
```

Comment:
Can we replace the `String.format` calls with the way slf4j does formatting? I think intellij might offer the option to replace the usage